src/estimators/any_sketch.py

```diff
@@ -317,8 +353,18 @@ def add(self, x):
     indexes = self.get_indexes(x)
     # Move this to the loop when we support more than one value function.
     value_function = self.config.value_functions[0]
+
+    if value_function == UniqueKeyFunction():
+      assert self.config.num_hashes == 1, (
+          'Same Key Aggregator only supports one hash function.')
+      assert isinstance(x, int) and (x >= 0) and (x < 2 ** 31), (
```

Comment:
Our hash function uses an int64. We may just want to udpate the dtype across the board for this.

Reply:
I'm keep int32 here because I add raw IDs as the keys (which are smaller than 2^31 ~ 2 billion).

If you prefer adding the hashed ID rather than raw ID, I will make the change!